app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -109,6 +109,16 @@
   </div>
   <%= render 'idv/doc_auth/start_over_or_cancel' %>
   <% unless Figaro.env.document_capture_react_enabled == 'false' %>
+    <%= nonced_javascript_tag do %>
+      <% asset_keys = [
```

Comment:
should we delete the `assets.js.erb` file and the update `asset_checker` to check this place instead?

Reply:
> should we delete the `assets.js.erb` file and the update `asset_checker` to check this place instead?

We could. My thinking was that this will be a temporary fix, and that eventually we'd want to restore `assets.js.erb`, or at least come up with some alternative approach which doesn't involve inlining scripts like this. It was more to get something in place for end-of-sprint to fix asset issues in the development sandbox.

That being said, to hold ourselves accountable, at the very least we'd want to track this via a ticket.

If it's simple enough a change, I can see how it'd be valuable to update the `asset_checker` script to check here, so that it's checking those strings as intended, especially given you'd just worked in #4114 to have this actually enforced 😅

Reply:
Working on making those changes, I see that #4113 should have updated the asset checker script, since we're no longer using `assetPath="example.png"` to refer to images, but rather `getAssetPath('example.png')`.

https://github.com/18F/identity-idp/blob/3602cf405582017cf651cb9dfbff66ff94733675/lib/asset_checker.rb#L25

Reply:
Oh great catch, thanks! I didn't even realize the way we referenced changed

Reply:
Updates in feea04a and d99ab44.